kalite/distributed/templates/distributed/base.html

```diff
@@ -82,9 +84,26 @@
 
         <script type="text/javascript" src="{% static 'js/fastclick.js' %}"></script>
         <script>
+
         $(function() {
             FastClick.attach(document.body);
         });
+
+         /* TODO(jtamiace): Backbonify this function and this page... when the time comes.
```

Comment:
@jtamiace Do you think you'll have time during the 0.14 cycle to tackle doing this? Wrapping the whole navbar into a Backbone view that is then toggled properly by changes in the statusModel would be great. This is also going to be helpful as part of the clean up required for the API driven login as well (at the moment we force a page refresh because our Javascript does not respond well to dynamic changes in the statusModel).